### PR TITLE
Fixes for AML block

### PR DESCRIPTION
**Pull request by tomer-shvadron**
### Description
Fixes for AML block and usages

### CODE REVIEW

File: apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx [hunk 1]
```diff
@@ -23,13 +24,9 @@ export const Navbar: FunctionComponent = () => {
         const isActiveFilterGroup = checkIsActiveFilterGroup(navItem);
 
         return (
-          <>
+          <Fragment key={`${navItem.key}-${isActiveFilterGroup}`}>
```
Comment (tomer-shvadron) [inline apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx:27]: Boyscouting, fixed `missing key` warning

Comment (MatanYadaev) [reply apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx:27]: Great. Thanks for the comment, it definitely reduces cognitive load.

**Review by MatanYadaev — APPROVED** [review]

File: apps/backoffice-v2/src/lib/blocks/variants/OngoingBlocks/hooks/useOngoingBlocksLogic/useOngoingBlocksLogic.tsx [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
     filterId: filterId ?? '',
   });
-  const kycSessionKeys = Object.keys(workflow?.context?.pluginsOutput?.kyc_session ?? {});
-  const getAmlData = useCallback((key: string) => workflow?.context?.aml, [workflow?.context?.aml]);
-  const amlBlock = useAmlBlock({
-    sessionKeys: kycSessionKeys,
-    getAmlData,
-  });
+
+  const amlData = useMemo(() => workflow?.context?.aml, [workflow?.context?.aml]);
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/variants/OngoingBlocks/hooks/useOngoingBlocksLogic/useOngoingBlocksLogic.tsx]: Redundant `useMemo`. Would make a bit more sense if it included the array wrapping.

File: apps/backoffice-v2/src/domains/workflows/fetchers.ts [hunk 3]
```diff
@@ -65,6 +66,7 @@ export const BaseWorkflowByIdSchema = z.object({
   workflowDefinition: WorkflowDefinitionByIdSchema,
   createdAt: z.string().datetime(),
   context: z.object({
+    aml: AmlSchema.optional(),
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/domains/workflows/fetchers.ts:69]: Greatly appreciated! 🙏🏼

File: apps/backoffice-v2/src/lib/blocks/components/AmlBlock/utils/aml-adapter.ts [hunk 4]
```diff
@@ -1,38 +1,34 @@
...
+  listingsRelatedToMatch: ListingRelatedToMatchSchema.optional().nullable(),
+});
+
+export const AmlSchema = z.object({
+  hits: z.array(HitSchema).optional().nullable(),
+  createdAt: z.string().optional().nullable(),
+  totalHits: z.number().optional().nullable(),
+});
+
+export const amlAdapter = (aml: z.infer<typeof AmlSchema>) => {
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/components/AmlBlock/utils/aml-adapter.ts]: Note in most cases `output` is probably what intended as it considers transforms done to the data.

```suggestion
export type TAml = z.output<typeof AmlSchema>;

export const amlAdapter = (aml: TAml) => {
```

Comment (Omri-Levy) [reply apps/backoffice-v2/src/lib/blocks/components/AmlBlock/utils/aml-adapter.ts]: `input` fits in the context of forms for example where the schema is used at different stages of the data.

**Review by Omri-Levy — APPROVED** [review]

File: apps/backoffice-v2/src/lib/blocks/variants/OngoingBlocks/hooks/useOngoingBlocksLogic/useOngoingBlocksLogic.tsx (hunk 2 above)
Comment (tomer-shvadron) [reply apps/backoffice-v2/src/lib/blocks/variants/OngoingBlocks/hooks/useOngoingBlocksLogic/useOngoingBlocksLogic.tsx]: You're right, I added the array into it.

**Pull request merged**